src/pallet/action.clj

```diff
@@ -226,7 +226,7 @@ of user options.
         (IllegalArgumentException.
          (format
           "Attempting to implement action %s with invalid dispatch value %s"
-          (action-symbol action) dispatch-val)))))
+          (action-symbol action))))))
```

Comment:
Format string has two placeholders, so should not `format` take two aruments (besides the format string)?